css/variables.scss

```diff
@@ -10,6 +10,7 @@ $react-dates-color-primary-shade-1: #33dacd !default;
 $react-dates-color-primary-shade-2: #66e2da !default;
 $react-dates-color-primary-shade-3: #80e8e0 !default;
 $react-dates-color-primary-shade-4: #b2f1ec !default;
+$react-dates-color-primary-shade-5: #ffe8bc !default;
```

Comment:
I think since this isn't a shade of `react-dates-color-primary` we should name this `react-dates-color-highlighted`